pkg/scaling/scale_handler.go

```diff
@@ -254,7 +264,9 @@ func (h *scaleHandler) checkScaledJobScalers(ctx context.Context, scalers []scal
 			scalerLogger.Info("Scaler is active")
 		}
 	}
-	maxValue = min(scaledJob.MaxReplicaCount(), devideWithCeil(queueLength, targetAverageValue))
+	if targetAverageValue != 0 {
```

Comment:
@TsuyoshiUshio could you please check, that this is ok?

Reply:
I added this judgment to avoid that if only the cpu/memory scaler is used in the ScalerJobs, the targetAverageValue is 0 (of course this is meaningless), which causes the devideWithCeil method to panic

Reply:
Hi @zroubalik @silenceper 
It looks good to me. TargetAverageValue is should be bigger than 1. In case of 0, that means no scale. So that This logic is correct. Thank you for adding the condition!

Reply:
Thanks @TsuyoshiUshio!